controllers/pipeline/pipeline_controller_test.go

```diff
@@ -110,7 +110,7 @@ var _ = Describe("PipelineController", func() {
 				CompletionTime: &metav1.Time{Time: now.Add(5 * time.Minute)},
 				TaskRunResults: []tektonv1beta1.TaskRunResult{
 					{
-						Name: "HACBS_TEST_OUTPUT",
+						Name: "TEST_OUTPUT",
```

Comment:
I saw a discussion about having it named as `APPSTUDIO_TEST_OUTPUT`

Reply:
The final decision was to have it as just `TEST_OUTPUT` (see [slack announcement](https://redhat-internal.slack.com/archives/C054SA6DWHK/p1682523721822739))